### PR TITLE
Add descriptions to core.

### PR DESCRIPTION
**Pull request by toneillbroad**
Dosage scores now return enum keywords.
Cleaned up gene and gene feature.

### CODE REVIEW

File: src/genegraph/source/graphql/core.clj [hunk 1]
```diff
@@ -91,7 +95,7 @@
                       :resolve resource/label
                       :description "Gene symbol"}
               :alternative_label {:type 'String
-                                  :resolve resource/alternative-label
+                                  :resolve gene/alternative-label
```
Comment (tnavatar) [inline src/genegraph/source/graphql/core.clj:98]: Is there a reason to prefer gene-specific selection of an alternative-label vs making one common to all resources?

File: src/genegraph/source/graphql/gene.clj [hunk 2]
```diff
@@ -72,59 +73,30 @@
...
-
-(defn chromosome [context args value]
-  (q/ld-> value []))
-
-(defn start-pos [context args value]
-  (q/ld-> value []))
-
-(defn end-pos [context args value]
-  (q/ld-> value []))
+(defn alternative-label [context args value]
```
Comment (tnavatar) [inline src/genegraph/source/graphql/gene.clj:98]: I think :skos/alternate-label isn't a property, you mean :skos/alternative-label? (which I think corresponds with the gene name)

File: src/genegraph/source/graphql/gene_feature.clj [hunk 3]
```diff
@@ -15,10 +15,10 @@
   )
 
 (defn previous-symbols [context args value]
-  )
+  (str/join ", " (q/ld-> value [:skos/hidden-label])))
 
 (defn alias-symbols [context args value]
-  (str/join ", " (q/ld-> value [:skos/hidden-label])))
```
Comment (tnavatar) [inline src/genegraph/source/graphql/gene_feature.clj:21]: Same as above, though I think I may have made a mistake in putting too much meaning into the SKOS terms by this point. Probably the right answer is to assign the aliases to a special attribute just for genes...

File: src/genegraph/source/graphql/core.clj (hunk 1 above)
Comment (toneillbroad) [reply src/genegraph/source/graphql/core.clj:98]: No, until now I didn't realize the ubiquity of the use of that property.

File: src/genegraph/source/graphql/gene.clj (hunk 2 above)
Comment (toneillbroad) [reply src/genegraph/source/graphql/gene.clj:98]: Reverted back to using the resource version so this is removed.

File: src/genegraph/source/graphql/gene_feature.clj (hunk 3 above)
Comment (toneillbroad) [reply src/genegraph/source/graphql/gene_feature.clj:21]: Help me understand the reasoning here... are you saying that skos/hidden-label casts too wide a net, and you feel that we should use one specific to aliases?

**Pull request merged**